operator/anarchyaction.py

```diff
@@ -117,6 +118,10 @@ def governor(self):
     def governor_name(self):
         return self.spec['governorRef']['name']
 
+    @property
+    def has_owner(self):
+        return True if self.metadata['ownerReferences'] else False
```

Comment:
I believe this needs to be `self.metadata.get('ownerReferences')` to avoid a python KeyError